featurehub-solid-sdk/src/hooks/useFeature.tsx

```diff
@@ -15,8 +16,12 @@ function useFeature<T = boolean>(key: string): Accessor<T> {
   const [value, setValue] = createSignal(client().feature<T>(key).value);
 
   createEffect(
-    on(client, () => {
-      if (listenerId) client().feature<T>(key).removeListener(listenerId);
+    on(ready, () => {
+      if (!ready()) return;
```

Comment:
Use new global `ready` signal to drive whether we subscribe to a feature key or not